panels/config/zwave/zwave-usercodes.html

```diff
@@ -46,11 +46,13 @@
             <div class='card-actions'>
               <paper-input
                 label='User code'
-                type=number
-                value='{{selectedUserCodeValue}}'
-                maxlength='{{userCodeMaxLen}}'
-                min='0'>
+                type=text
```

Comment:
you're a bit sloppy when it comes to adding quotes around HTML attributes. It's not necessary perse as browsers can handle it. However, it makes it easier to make mistakes (ie a space in the value will cut off the value at that point).

All attributes in HTML require to be wrapped in quotes.

Reply:
Fixed, still learning :+1: